platforms/spring-boot/components-starter/camel-ahc-starter/src/main/java/org/apache/camel/component/ahc/springboot/AhcComponentAutoConfiguration.java

```diff
@@ -110,4 +110,4 @@ private boolean isEnabled(
             return resolver.getProperty("enabled", Boolean.class, defaultValue);
         }
     }
-}
```

Comment:
yeah hard to see, a newline was added at the end of the file here: https://github.com/apache/camel/pull/1670/commits/dd6d00d339b8b0f1c7454bff454d8aa0ce00a786#diff-6f17f35d925830ab6967b522fe0eb81cR155